### ISSUE TITLE
Game balance - September 2015

### ISSUE THREAD

**Issue by Phrancis** [Cardshifter/Cardshifter#382]
Please post here any game balance issues/concerns you find so they can be addressed/fixed. Please make sure to identify the Mod, the Card(s) in question, and what you think is unbalanced.

**Comment by Zomis** [comment]
**Cyborg Chronicles**

I am not sure if the solution to #169 is the best one, we might want to re-think that. One possible solution might be to allow scrapping cards when they are at your hand (without paying their mana cost first, and without having to wait for them to be able to attack).

The game mechanics makes it extremely hard for a player to turn the game around when he is already losing. Try having the creatures heal *at the beginning* of your turn instead of at the end, this will allow your opponent to attack them while they have weaker health.

The whole "all cards have taunt" mechanism is also a contributing factor to the issue that if player A has many creatures in play, and player B wants to play a creature but if he would then player A would just kill the creature and at the end of the turn have his creatures healed again.

Overall, more cards are needed (spells especially, which can deal damage or destroy opponent cards)

These comments are from two games when I and a friend played, many of these comments are from my friend.

**Comment by Etrisa** [comment]
Quick Intruduction:
Hello, I call myself Etrisa and I'm a big fan of games, playing them, discussing them, creating them.
I'm Zomis younger cousin and Tejpbits big brother.
I'm an excellent bug tester and often a good person to discuss things with.
TL:DR will often be found at the top and/or bottom of my lengthier posts.

After playing both Mythos and Cyborg-Chronicles I've realized one thing: There's nothing that a simple Lightning bolt can't solve. http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=205227&type=card

I played all of the games VS Zomis to test things out and had a blast doing so.

From both Mythos and Cyborg I feel a lack of removal spells.
That Spy(Mythos) would be a lot less sneaky if i dealt 4 dmg to it and got rid of that amazing yet annoying -1Atk/-1Health to all opponent creatures ability.
Same thing goes for Cyborg, if i could sacrifice a creature FROM MY HAND for scrap instead of having to play it that would hopefully make the game more all around balanced. Cyborg felt very spiky. Once you're ahead it feels like you'll stay ahead (unless you do a major screw up). Allowing me to sacrifice a creature for scrap from my hand would allow me to play a ATK heavy creature and then buff it with enchantments thus forcing my opponent to deal with it and hopefully use more than 1 of his creatures to do so.
Possibly add some more enchantments to that.
For example:

Name:Broken Power Outlet
Cost: 3 Scrap
Enchant target Creautre, that creature dies.
Flavor text: Always put pointy stuff into power outlets!

Overall i liked Cardshifter though. I can't wait for it to become a little more clear/easy to use so i can battle it out again! Hopefully i can add something to the game myself in terms of cards and feedback.

As promised, here is the TL:DR:
Mythos:
Everything that changes the attack/health on creatures felt a bit to powerful.
Soloution: Add more cards! It's as simple as that, more options often leads to a more diverse game/match.

Cyborg:
It felt one-sided more often than not.
I think most of the issues can be solved by adding more cards to the game.

**Comment by Phrancis** [comment]
Hello @Etrisa and thank you for the feedback! 

Mythos badly needs so balancing, I've noticed a lot of the same things as you have, it's not unusual to see cards buffed to really high numbers like 9/12 when multiple buffs stack together. We're going to go over all of Mythos again, making adjustments. If you notice anything specific, you're welcome to mention it here!

For Cyborg: The ability to sacrifice a card for scrap from the hand is a good idea that we should support. Right now the only workaround to this is to set `sickness 0` on a card, effectively giving it Rush, as the sickness affects the action of Scrapping as well as attacking. 

@Zomis correct me if I'm wrong but didn't we originally have it to where you could scrap any card which can be scrapped as soon as it entered the Battlefield? I realize it's not the same thing as "playing from hand" of course. 

For playing from hand, we would need to change the actions structure a bit to where a Scrap action would show on cards in your hand as well, granted you have the Mana to actually "play" the card so they can be scrapped. @Etrisa was that the intention?

**Comment by jacwah** [comment]
In addition to spells that deal direct damage to creatures, Hearthstone also has other ways for players that are slightly behind to deal with powerful creatures. There are cards that can be used to "silence" creatures, removing all their special effects and enchantments. There's also cards like [Polymorph](http://www.hearthpwn.com/cards/595-polymorph) and [Shadow word: Death](http://www.hearthpwn.com/cards/547-shadow-word-death) that can nullify any very powerful creature. These effects could possibly help with the "spikyness" of the games.

**Comment by Etrisa** [comment]
@Phrancis 
> For playing from hand, we would need to change the actions structure a bit to where a Scrap action would show on cards in your hand as well, granted you have the Mana to actually "play" the card so they can be scrapped. @Etrisa was that the intention?

Almost, only you would not have to pay the many cost in order to scrap the card.
I do realize that this opens up the possibility to scrap an "infinite" number of cards, witch might be too powerful. Then again you could put a limit to this saying you can only scrap one card each turn, or possibly you can't have more than X scrap in total. Or just leave it be and see what happens, the possibilities are as always endless. Possibly add an extra ability that says if you scrap the card Y happens. Where Y might be, draw a card, put a 1/1 with Haste onto the battlefield, or possibly a drawback if you get a lot of scrap, say you get 5 or 6 scrap from a card you might want a drawback so if you scrap it you'll have to spend mana/discard a card/discard random card etc etc etc.

The best way is probably just to throw it in, playtest it and see how it feels after a few games.

**Comment by Zomis** [comment]
I don't think we need any limit to how many cards you can scrap each turn, except for a limit that you can only scrap cards that you have on hand ;) I like the ideas of having triggers for when you scrap a card.

**Comment by Lokkij** [comment]
As @Phrancis noted in the chatroom, the boardstates in Mythos become too complex as the game progresses. I've come up with a couple of solutions to this problem:

- **Less continuous effects, more permanent/timed effects**. A lot of cards currently modify all creatures' stats as long as they are on the battlefield. Changing those effects to either a permanent effect upon entering the battlefield or an until-end-of-turn effect reduces board complexity massively: these just happen and the player then doesn't have to think about them very much, in contrast to continuous effects.
- **Less random effects**. Random effects greatly increase the number of possible future boardstates when thinking ahead: the difference between healing for 3 and dealing 3 damage at the end of the turn can mean a completely different optimal strategy for the turn. In addition, these kinds of random effects are also considered unfun by many players as they feel it gives them less control over the game. The Great Mark Rosewater has written [an article about randomness](http://archive.wizards.com/Magic/magazine/article.aspx?x=mtg/daily/mm/37) that's definitely worth reading if we want to keep some of the random effects.

I'll probably think of more as I test more and I'll post my findings here when that happens.
